hack/run-locally.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash 
+
+
+
+set -o errexit
+set -o nounset
+
+# Install our overrides so the cluster doesn't run the network operator.
+function override() {
+    if [[ ! -e ${CLUSTER_DIR}/manifests/cvo-overrides.yaml ]]; then
+        echo "cannot find cvo-overrides.yaml; please run"
+        echo "openshift-install --dir=${CLUSTER_DIR} create manifests"
+        exit 1
+    fi
+
+    # Patch the CVO to not create the network operator
+    echo "Applying overrides to ${CLUSTER_DIR}/manifests/cvo-overrides.yaml"
+
+    kubectl --kubeconfig=hack/null-kubeconfig patch --type=json --local=true -f=${CLUSTER_DIR}/manifests/cvo-overrides.yaml -p "$(cat hack/overrides-patch.yaml)" -o yaml > ${CLUSTER_DIR}/manifests/.cvo-overrides.yaml.new
```

Comment:
Since we're requiring `oc` elsewhere, change this to `oc` for consistency?

Reply:
For some reason this doesn't work with `oc`. I don't remember why.